tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -302,6 +314,24 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string TraceMethods = "DD_TRACE_METHODS";
 
+        /// <summary>
+        /// Configuration key for specifying a custom regex to obfuscate query strings.
+        /// Default value is in TracerSettings
+        /// </summary>
+        public const string ObfuscationQueryStringRegex = "DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP";
+
+        /// <summary>
+        /// Configuration key for specifying a timeout in milliseconds to the execution of the query string obfuscation regex
+        /// Default value is 100ms
```

Comment:
Is there a desire to standardize the default timeout across tracers? I didn't see a standardized value for this

Reply:
nope there isn't yet.. 100ms seemed reasonable , will add a comment in the RFC about it